src/GeneralStateTestsFiller/stBadOpcode/badOpcodesFiller.json

```diff
@@ -30,49 +30,138 @@
                 "nonce": "0",
                 "storage": {
                 }
+            },
+            "1000000000000000000000000000000000000001" : {
+                "balance" : "1000000000000000000",
+                "code": "0x",
+                "nonce": "0",
+                "storage": {
+                }
             }
         },
         "transaction" : {
             "data" : [
-		"0x60016001600160010c",
-		"0x60016001600160010d",
-		"0x60016001600160010e",
-		"0x60016001600160010f",
-		"0x60016001600160011b",
-		"0x60016001600160011c",
-		"0x60016001600160011d",
-		"0x60016001600160011e",
-		"0x60016001600160011f",
-		"0x600160016001600121",
-		"0x600160016001600122",
-		"0x600160016001600123",
-		"0x600160016001600124",
-		"0x600160016001600125",
-		"0x600160016001600126",
-		"0x600160016001600127",
-		"0x600160016001600128",
-		"0x600160016001600129",
-		"0x60016001600160012a",
-		"0x60016001600160012b",
-		"0x60016001600160012c",
-		"0x60016001600160012d",
-		"0x60016001600160012e",
-		"0x60016001600160012f",
-		"0x60016001600160013f",
-		"0x600160016001600146",
-		"0x600160016001600147",
-		"0x600160016001600148",
-		"0x600160016001600149",
-		"0x60016001600160014a",
-		"0x60016001600160014b",
-		"0x60016001600160014c",
-		"0x60016001600160014d",
-		"0x60016001600160014e",
-		"0x60016001600160014f",
-		"0x60016001600160015c",
-		"0x60016001600160015d",
-		"0x60016001600160015e",
-		"0x60016001600160015f"
+"0x60016001600160010c60006000f3",
```

Comment:
you could add one valid opcode to check that a new contract is actually being created this way

Reply:
Is there a way to do this without adding an expect section for each index in the data array?  I am only familiar with the `"field": "-1" ` syntax which applies to all indices.  

I'm not sure how to make the expect section different for a single case (or a subset of cases).
^Maybe this could be a useful feature?

Reply:
Okay.  just discovered array syntax `"data" : [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16 ... ]`.

Reply:
this is also true for network field